Bottom: 49ccd52cec5c6bca412f099b8e982ef73efc28e9
Top:    d29b829f9339dc93a5bdc8117f0a5e8d963291bf
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-23 14:41:55 +0100

Fix last commit to work with new crosshair notification API


---

diff --git a/src/action.c b/src/action.c
index 7adb9de..4181f2e 100644
--- a/src/action.c
+++ b/src/action.c
@@ -4847,7 +4847,6 @@ ActionToggleStripHierarchy (int argc, char **argv, int x, int y)
   char *function = ARG (0);
   if (function && PCB->ElementOn)
     {
-      HideCrosshair (true);
       switch (GetFunctionID (function))
 	{
 	case F_Object:
@@ -4897,7 +4896,6 @@ ActionToggleStripHierarchy (int argc, char **argv, int x, int y)
 	      }
 	  }
 	}
-      RestoreCrosshair (true);
     }
   return 0;
 }
